.github/workflows/test_examples_small.yml

```diff
@@ -9,12 +9,18 @@ env:
 jobs:
   test:
     runs-on: ${{ matrix.platform }}
-    name: "${{ matrix.platform }} ${{ matrix.toolchain }}"
+    name: "${{ matrix.example }} - ${{ matrix.platform }}"
     strategy:
+      fail-fast: false
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        toolchain: [1.81]
-        # toolchain: [stable]
+        toolchain: [stable]
+        example: [
```

Comment:
this list can be potentially joined into `test_examples.yml` so that there's only one examples related job